firmwares/heatit/z-trm6.json

```diff
@@ -0,0 +1,25 @@
+{
+  "devices": [
+    {
+      "brand": "Heatit",
+      "model": "Z-TRM6",
+      "manufacturerId": "0x019b",
+      "productType": "0x0030",
+      "productId": "0x3001"
+    }
+  ],
+  "upgrades": [
+    {
+      "version": "1.0.11",
```

Comment:
Is there any limitation on upgrade paths? Are there versions that may not be updated or downgraded to this version?
If not, this is fine.

Reply:
Ok, then we need to keep that in mind when there is a future update. The applicable updates can be limited like this
```
"$if": "firmwareVersion < 1.0.11",
```
for example.